content/contact.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Contact"
+date: 2017-07-03T14:28:34-04:00
+draft: false
+mainpage: false
+projectpage: false
+projectslug: ""
+projectname: ""
+listdescription: ""
+---
+
+### Contact
+If you would like to reach out to the NASA WorldWind team, please try the following:
```

Comment:
Let's drop this text. I think it's superfluous. My impression of good modern web design is that sites omit text that is redundant. For example, we'd omit text that's made redundant by the page's context, the page headings, and the page layout.